src/main/webapp/components/faceted-dropdown/index.tsx

```diff
@@ -56,7 +56,9 @@ const FacetedDropdown = (props: Props) => {
     )
   }
 
-  const facetEnums = getIn(data, ['facet'], []).map((facet: any) => facet.value)
+  const facetEnums = (getIn(data, ['facet'], undefined) || []).map(
```

Comment:
data.facet may be unset or null - in the latter case, getIn won't default to an empty array